adr-0001-adr-template.md

```diff
@@ -0,0 +1,17 @@
+# ADR 0001: ADR Template
```

Comment:
Shall we have a scope in the filename? 
E.g. 
- adr-0001-clients-javascript-[...].md
- adr-0001-core-[...].md
- adr-0001-tenderdash-[...].md